src/main/java/com/gluonhq/gradle/attach/AttachConfiguration.java

```diff
@@ -117,38 +125,28 @@ private void applyConfiguration() {
         if (services != null) {
             services.forEach(serviceDefinition -> project.getDependencies()
                             .add(configuration.getName(), generateDependencyNotation(configuration, serviceDefinition, target)));
-
-            Map<String, String> utilDependencyNotationMap = new HashMap<>();
-            utilDependencyNotationMap.put("group", AttachResolver.DEPENDENCY_GROUP);
-            utilDependencyNotationMap.put("name", AttachResolver.UTIL_ARTIFACT);
-            utilDependencyNotationMap.put("version", getVersion());
-            project.getDependencies().add(configuration.getName(), utilDependencyNotationMap);
```

Comment:
This is still required. It allows adding `util` by default, without the need of an explicit dependency.
You can also deal here with the Android exception:

```
Map<String, String> utilDependencyNotationMap = new HashMap<>();
            utilDependencyNotationMap.put("group", DEPENDENCY_GROUP);
            utilDependencyNotationMap.put("name", UTIL_ARTIFACT);
            utilDependencyNotationMap.put("version", getVersion());
            if (Constants.PROFILE_ANDROID.equals(target)) {
                utilDependencyNotationMap.put("classifier", target);
            }
            project.getDependencies().add(configuration.getName(), utilDependencyNotationMap);

Reply:
Ah, so that was it. I could not grok that code. Get it know. This resolves the question I asked you.